demisto_sdk/commands/doc_reviewer/doc_reviewer.py

```diff
@@ -101,7 +107,9 @@ def get_files_from_git(self):
         click.secho('Gathering all changed files from git', fg='bright_cyan')
         for file in self.gather_all_changed_files():
             file = str(file)
-            if os.path.isfile(file) and find_type(file) in self.SUPPORTED_FILE_TYPES:
+            if os.path.isfile(file) and find_type(
+                file, ignore_invalid_schema_file=self.ignore_invalid_schema_file
+            ) in self.SUPPORTED_FILE_TYPES:
```

Comment:
let's try to cover these lines as well, you can mock the gather_all_changed_files if needed

Reply:
done